src/config.rs

```diff
@@ -24,6 +26,10 @@ pub struct Config {
     pub subprocesses: Vec<SubProcess>,
 }
 
+fn is_true() -> bool {
+    true
+}
+
```

Comment:
You are right, I forgot the `default` but I don't like this.
My idea here was
* I set it on `true` when creating the config file.
* The user keep it without touching at the file
* the user remove it by
  * removing the line
  * setting it on `false`
So we can maybe stay on `false` by default since the user need to interact for removing the message
My problem here is that new users will have it by default, people who already have a config file will not

Reply:
Can they remove it by removing the line though? I think it will just fail to parse if the line is not there. That's the point of the `default` right? Note that you didn't have the decorator when I commented: https://github.com/yozhgoor/cargo-temp/pull/70#discussion_r968265992

Reply:
>  My problem here is that new users will have it by default, people who already have a config file will not

That sounds good right? I think you made the right choice

Reply:
> I think it will just fail to parse if the line is not there. That's the point of the `default` right? Note that you didn't have the decorator when I commented

Yeah sorry, that's what I wanted to say. I forgot the `default` so removing the lines would have fail but the `default = "is_true"` means that if you delete the line the welcome message will still be there and I didn't like that.
In this case the default value is not really the same as the "unset" value